src/renderer/extensions/vueNodes/components/LGraphNode.vue

```diff
@@ -353,6 +354,10 @@ const handlePointerDown = (event: PointerEvent) => {
 
   // Start drag using layout system
   isDragging.value = true
+
+  // Set Vue node dragging state for selection toolbox
+  layoutStore.setVueNodeDraggingState(true)
```

Comment:
What's the difference between isDragging and layoutStore.isDraggingVueNodes?

Reply:
When would they have different values?
Could we eliminate the per-node one and instead of having a boolean ref have the node that's being dragged's ID as both an indicator that dragging is occurring and a source of truth for which node is being dragged?